src/Analysis/Ast/Impl/Modules/PythonPackage.cs

```diff
@@ -25,28 +27,54 @@ namespace Microsoft.Python.Analysis.Modules {
     /// Represents package with child modules. Typically
     /// used in scenarios such as 'import a.b.c'.
     /// </summary>
-    internal sealed class PythonPackage : PythonModule, IPythonPackage {
-        private readonly ConcurrentDictionary<string, IPythonModule> _childModules = new ConcurrentDictionary<string, IPythonModule>();
+    internal sealed class PythonVariableModule : IPythonModule, IEquatable<IPythonModule> {
```

Comment:
Didn't like the `package` name? :-)

Reply:
It isn't a package anymore. I actually don't know how to call it better. Need to update the comment btw.